src/main/java/com/twingly/search/Constants.java

```diff
@@ -12,7 +12,12 @@
     /**
      * The constant DATE_FORMAT.
      */
-    public static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
+    public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssX";
```

Comment:
@roback nope.

`Z` in Java is for RFC standard only. And `X` is for ISO standard.